deploy/chart/devfile-registry/templates/_template.tpl

```diff
@@ -17,6 +17,20 @@
 {{- .Values.hostnameOverride | default (printf "devfile-registry-%s" .Release.Namespace) -}}
 {{- end -}}
 
+{{- define "devfileregistry.ingressHostname" -}}
+{{- $hostname := .Values.hostnameOverride | default (printf "devfile-registry-%s" .Release.Namespace) -}}
+{{- .Values.global.ingress.domain | printf "%s.%s" $hostname -}}
```

Comment:
Will this format differ than the one from https://github.com/devfile/registry-operator/pull/80? On the operator side the hostname is `{hostName}.{ingressDomain}` but here it looks like `{ingressDomain}.{hostName}`. Is this the case and if so does it matter or should they be identical in formatting?

Reply:
@Jdubrick The piping `{{- .Values.global.ingress.domain | printf "%s.%s" $hostname -}}` passes `.Values.global.ingress.domain` into the last argument of `printf`, so its still `{hostName}.{ingressDomain}`. I guess I could do `{{- printf "%s.%s" $hostname .Values.global.ingress.domain -}}` instead since the piping seems unnecessary.

Reply:
Ah I was under the impression it was going first, in that case I think it's all good, the piping gets the job done I just misunderstood.